### DISCUSSION TITLE
Mise en place d'une coche permettant aux publics en immersion d'autoriser les établissements d'accueil à avoir leur adresse mail

### DISCUSSION THREAD

**Discussion by MagaliMoserNoriaIRL** [unistra/esup-immersup-demandes#14] [7- Autres sujets]
Demande de l'EM Strasbourg 
Mise en place d'un système de coche pour les personnes venant en immersion pour autoriser l'établissement d'accueil à avoir son adresse mail pour pouvoir le contacter personnellement en dehors de la plateforme.

**Comment by efinkbeiner5** [comment]
Attente avis DPO
- Aujourd'hui, ils peuvent avoir une liste de diffusion des personnes ayant fait une immersion au sein de leur structure, durant la durée de l'année mais pas après.

**Comment by efinkbeiner5** [comment]
Refus DPO, immersup ne sert pas à faire un annuaire de prospect. Ils peuvent leur envoyer un mail les invitant à se déclarer sur une autre plateforme amis aucun transfert automatique.
